lib/simulator.js

```diff
@@ -14,9 +14,6 @@ function handleUnsupportedXcode (xcodeVersion) {
     throw new Error(`Tried to use an iOS simulator with xcode ` +
                     `version ${xcodeVersion.versionString} but only Xcode version ` +
                     `6.0.0 and up are supported`);
-  } else if (xcodeVersion.major >= 10) {
-    throw new Error(`Xcode version ${xcodeVersion.versionString} is ` +
```

Comment:
Can we print a warning message instead?

Reply:
About the compatibility with Xcode version X has not been tested for this Appium version, so one might expect issues.

Reply:
Xcode 5 will not work with Appium. So an exception is warranted.

Reply:
I'm talking about the case if Xcode version is greater than the last known one, but in case you'd like to keep it as is then I don't mind

Reply:
That exception is gone here. A new Xcode major version will just get the latest known version.